### ISSUE TITLE
AOT Compilation failing in Angular 5

### ISSUE THREAD

**Issue by zatomik** [emn178/angular2-chartjs#27]
Hi, getting the following when running an AOT build, in angular 5.01, and cli 1.50:

```
ERROR in ./node_modules/angular2-chartjs/dist/chart.component.ngfactory.ts
Module build failed: Error: /home/xxx/node_modules/angular2-chartjs/dist/chart.component.ngfactory.ts is not part of the compilation. Please make sure it is in your tsconfig via the 'files' or 'include' property.
```

I've tried adding the lib to include in `tsconfig.app.json` as a workaround as mentioned in these issues: 
[8263](https://github.com/angular/angular-cli/issues/8263)
[8284](https://github.com/angular/angular-cli/issues/8284) but have not had success,

Any advice or workarounds? thanks

**Comment by mafalarz** [comment]
Adding symlinks: true to the resolve section of my webpack.config.js fixed the issue for a custom library we built. Not sure if it will work chartjs, but thought I'd share

```
var config = {};
config.resolve = {
    extensions: ['.ts', '.js'],
    symlinks: true,
    modules: ['./node_modules']
};
...etc
module.exports = config;
```

**Comment by emn178** [comment]
v0.4.1 should fix this issue. please try again.

**Comment by zatomik** [comment]
confirm issue fixed in v0.4.1, thank you
